ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -55,6 +71,23 @@ QObject {
         return networkString
     }
 
+    function getAllChainIds() {
```

Comment:
This should ideally be in some `Utils`; not the first time I'm seeing this code being copy/pasted around